File: Anul I - Licenta/Semestrul I/Programarea Algoritmilor/Seminare/Seminarul 6/PP03. Turnurile din Hanoi/hanoi.cpp
// https://www.geeksforgeeks.org/problems/tower-of-hanoi-1587115621/1?itm_source=geeksforgeeks

class Solution {
  public:
    int towerOfHanoi(int n, int from, int to, int aux) {
        if (n == 0)
            return 0;
        else
        {
            return 1 + towerOfHanoi(n - 1, to, aux, from)
                     + towerOfHanoi(n - 1, aux, to, from);
        }
    }
};

File: Anul I - Licenta/Semestrul I/Programarea Algoritmilor/Seminare/Seminarul 6/PP03. Turnurile din Hanoi/hanoi.py
# https://www.geeksforgeeks.org/problems/tower-of-hanoi-1587115621/1?itm_source=geeksforgeeks

class Solution:
    def towerOfHanoi(self, n, fromm, to, aux):
        if (n == 0):
            return 0
        else:
            return 1 + self.towerOfHanoi(n - 1, to, aux, fromm) + self.towerOfHanoi(n - 1, aux, to, fromm)